src/Health.js

```diff
@@ -0,0 +1,21 @@
+class Health {
+  constructor(data) {
+    this.data = data
+  }
+
+  calculateDailyTotal(id, date, property) {
+    let findDataByDate = this.data.find((data) => id === data.userID && date === data.date);
+    return findDataByDate[property];
+  }
+
+  calculateAverage(id, property) {
+    let dataPerDay = this.data.filter((data) => id === data.userID);
+    let total = dataPerDay.reduce((sum, data) => {
+      return sum += data[property];
+    }, 0);
+    let averagePerDay = total / dataPerDay.length;
+    return averagePerDay;
+  }
+}
+
+export default Health;
```

Comment:
Nice! cool to see inheritance of this new Parent class working clean up our redundancies!
Well done, Coop!